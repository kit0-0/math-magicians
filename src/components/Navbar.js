import React from 'react';
import { Link } from 'react-router-dom';

function Navbar() {
  return (
    <nav className="bg-gray-800">
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
        <div className="flex items-center justify-between h-16">
          <div className="flex items-center">
            <Link to="/" className="text-white text-2xl font-bold mr-4">
              <h1>Math Magicians</h1>
            </Link>
          </div>
          <div className="flex items-center">
            <Link to="/" className="text-white hover:text-gray-300 ml-4">
              Home
            </Link>
            <Link to="/calculator" className="text-white hover:text-gray-300 ml-4">
              Calculator
            </Link>
            <Link to="/quote" className="text-white hover:text-gray-300 ml-4">
              Quote
            </Link>
          </div>
        </div>
      </div>
    </nav>
  );
}

export default Navbar;
