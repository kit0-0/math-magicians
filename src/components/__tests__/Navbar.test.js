import React from 'react';
import renderWithRouter from '../test-utils';
import Navbar from '../Navbar';
import '@testing-library/jest-dom/extend-expect';

describe('Navbar component', () => {
  it('should match the snapshot', () => {
    const { container } = renderWithRouter(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
