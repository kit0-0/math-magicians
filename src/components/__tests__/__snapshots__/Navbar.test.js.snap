// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component should match the snapshot 1`] = `
<div>
  <nav
    class="bg-gray-800"
  >
    <div
      class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
    >
      <div
        class="flex items-center justify-between h-16"
      >
        <div
          class="flex items-center"
        >
          <a
            class="text-white text-2xl font-bold mr-4"
            href="/"
          >
            <h1>
              Math Magicians
            </h1>
          </a>
        </div>
        <div
          class="flex items-center"
        >
          <a
            class="text-white hover:text-gray-300 ml-4"
            href="/"
          >
            Home
          </a>
          <a
            class="text-white hover:text-gray-300 ml-4"
            href="/calculator"
          >
            Calculator
          </a>
          <a
            class="text-white hover:text-gray-300 ml-4"
            href="/quote"
          >
            Quote
          </a>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
